config/application.yml.example

```diff
@@ -62,3 +62,6 @@ SMTP_PASSWORD: 'f00d'
 # STRIPE_INSTANCE_PUBLISHABLE_KEY: "pk_test_xxxx" # This can be a test key or a live key
 # STRIPE_CLIENT_ID: "ca_xxxx" # This can be a development ID or a production ID
 # STRIPE_ENDPOINT_SECRET: "whsec_xxxx"
+
+# URLs for the site's main menu
+# MENU_3_URL: https://about.openfoodnetwork.org.uk/services
```

Comment:
is this mandatory? what happens if this is not configured?

Reply:
![Screenshot from 2019-04-05 13-03-04](https://user-images.githubusercontent.com/762088/55623538-96a21080-57a3-11e9-885d-8b544cbb98b0.png)

I let RSpec answer you :sunglasses: I'll open a PR on ofn-install to add the required ENV vars for each instance once I get your approvals.